circleguard/loadable.py

```diff
@@ -542,6 +544,24 @@ def __repr__(self):
     def __str__(self):
         return f"Replay by {self.username} on {self.map_id}"
 
+    def as_arrays(self):
```

Comment:
would prefer this logic to happen in `__init__`; it's 6 extra lines but nothing too complicated. And the logic only happens once so we shouldn't be making it available as a function.